smartseq2_single_sample/BuildRef/BuildReferenceBundle.wdl

```diff
@@ -0,0 +1,63 @@
+task BuildStarReference{
+  File ref_fasta
+  File gtf_file
+  
+  command {
+    mkdir star
+    STAR --runMode genomeGenerate \
+      --genomeDir star \
+      --genomeFastaFiles ${ref_fasta} \
+      --sjdbGTFfile ${gtf_file} \
+      --sjdbOverhang 100 \
+      --runThreadN 16
+    tar -cvf star.tar star
+  }
+  output {
+    File starRef = "star.tar"
+  }
+  runtime {
+    docker:"humancellatlas/star_dev:v1"
+    memory: "50 GB"
+    disks :"local-disk 100 HDD"
+    cpu:"16"
+  }
+}
+
+task BuildRsemReference {
+  File ref_fasta
+  File gtf_file
+
+  command {
+    mkdir rsem
+    rsem-prepare-reference --gtf ${gtf_file} --bowtie ${ref_fasta} rsem/rsem_trans_index
+    tar -cvf rsem.tar rsem/
+  }
+  output {
+    File rsemRef= "rsem.tar"
+  }
+  runtime {
+    docker: "humancellatlas/rsem"
+    memory: "10 GB"
+    disks: "local-disk 100 HDD"
+  }
+}
+
+workflow BuildRef {
+  File fasta
+  File gtf
+  
+  call BuildStarReference {
+    input:
+      ref_fasta = fasta,
+      gtf_file = gtf
+  }
+  call BuildRsemReference{
```

Comment:
Inconsistency in formatting: below the inputs are input=value, above they are input = value.

Reply:
I find the extra spaces a bit clearer, input = value. We need that linter.